lib/metrics/metrics-tls-cache.c

```diff
@@ -84,15 +84,15 @@ _deinit_tls_clusters_map_thread_init_hook(gpointer user_data)
 }
 
 static void
-_init_tls_clusters_map_apphook(gint type, gpointer user_data)
+_init_tls_clusters_map_apphook(void)
```

Comment:
If I understand correctly, this is not an apphook anymore, so maybe remove that from the name.

Reply:
Thanks, fixed.